inst/tests/tests.Rraw

```diff
@@ -10009,9 +10009,9 @@ if (!haszlib()) {
 } else {
   test(1658.41, fwrite(data.table(a=c(1:3), b=c(1:3)), compress="gzip"), output='a,b\n1,1\n2,2\n3,3')  # compress ignored on console
   DT = data.table(a=rep(1:2,each=100), b=rep(1:4,each=25))
-  test(1658.421, fwrite(DT, file=f1<-tempfile(fileext=".gz"), verbose=TRUE), NULL,
+  test(1658.421, filepath <- fwrite(DT, file=f1<-tempfile(fileext=".gz"), verbose=TRUE), filepath,
```

Comment:
is `filepath` different from `f1`?

Ditto for `f2` below.

Reply:
done 👍